lib/cluster.js

```diff
@@ -57,11 +57,11 @@ function Worker(options) {
 }
 util.inherits(Worker, EventEmitter);
 
-Worker.prototype.kill = function() {
+Worker.prototype.kill = function kill() {
   this.destroy.apply(this, arguments);
 };
 
-Worker.prototype.send = function() {
+Worker.prototype.send = function send() {
```

Comment:
doesn't v8 already know the names of functions assigned to named prototype properties?